jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -730,215 +730,6 @@ properties:
             type: number
             fits_keyword: PA_APER
             fits_hdu: SCI
```

Comment:
We also have to move out the other few WCS-related keywords that are now assigned to the SCI header, which includes RADESYS, PA_APER, VA_SCALE, RA_V1, DEC_V1, PA_V3 (I think that's all of them). Basically just look for any keyword the has "fits_hdu: SCI", even if they aren't part of the wcsinfo block.